src/GafferSceneUIModule/HierarchyViewBinding.cpp

```diff
@@ -510,8 +510,8 @@ class InclusionsColumn : public PathColumn
 
 		IE_CORE_DECLAREMEMBERPTR( InclusionsColumn )
 
-		InclusionsColumn( ContextPtr context )
-			:	PathColumn(), m_context( context )
+		InclusionsColumn( ContextPtr context, unsigned flags = PathColumn::ResizeToContents )
```

Comment:
Is there any real benefit to exposing this as a constructor argument? Perhaps InclusionsColumn knows best and should just hardcode itself to ResizeToContents?